File: Linguagem C++/Estrutura Repetitiva/ParImpar.cpp
#include <iostream>

using namespace std;

int main()
{
    int n,x;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    for (int i=1;i<=n;i++){

        cout << "Digite um numero: ";
        cin >> x;

        if (x%2==0 && x>0){
            cout << "PAR POSITIVO" << endl;
        }else if (x%2==0 && x<0){
            cout << "PAR NEGATIVO" << endl;
        }else if(x%2!=0 && x>0){
            cout << "IMPAR POSITIVO" << endl;
        }else if (x%2!=0 && x<0){
            cout << "IMPAR NEGATIVO" << endl;
        }else{
            cout << "NULO" << endl;
        };
    };

    return 0;
}

File: Linguagem Java/Exercicios/ParImpar.java
import java.util.Locale;
import java.util.Scanner;

public class ParImpar {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n,x;
		
		System.out.print("Quantos numeros voce vai digitar? ");
        n = sc.nextInt();
        
        for (int i=1;i<=n;i++) {
        	System.out.print("Digite um numero: ");
        	x = sc.nextInt();
        	
        	if (x % 2 == 0 && x > 0) {
        		System.out.println("PAR POSITIVO");        		
        	}else if (x % 2 == 0 && x < 0) {
        		System.out.println("PAR NEGATIVO");
        	}else if (x % 2 != 0 && x > 0) {
        		System.out.println("IMPAR POSITIVO");        		
        	}else if (x % 2 != 0 && x < 0) {
        		System.out.println("IMPAR NEGATIVO");        		
        	}else {
        		System.out.println("NULO");        		
        	};
        };
        
        sc.close();
	}

}
